spec/acceptance/self_service_spec.rb

```diff
@@ -149,6 +149,10 @@ class {'self_service':
         expect(result.stdout).to match(%r{false})
         run_shell('puppet config set use_cached_catalog false', expect_failures: false)
       end
+      it 'if S0039 conditions for false are met' do
+        result = run_shell('facter -p self_service.S0039')
```

Comment:
You dont have any conditions to make the test fail, the 503 line will not exist in the test system so your test can never pass

Reply:
I have updated the readme and code as requested